hq/main.py

```diff
@@ -0,0 +1,20 @@
+# В модуль с проверкой даты добавьте возможность запуска в терминале с передачей даты на проверку.
+
+import sys
+from datetime import datetime
+
+def check_date(date_str):
+    try:
+        date = datetime.strptime(date_str, "%Y-%m-%d")
```

Comment:
А переменная то вам зачем?
```suggestion
        datetime.strptime(date_str, "%Y-%m-%d")
```